### ISSUE TITLE
TypeError: Cannot read property 'val' of undefined

### ISSUE THREAD

**Issue by agross** [WLAN-Kabel/ioBroker.calendar#29]
**Describe the bug**  

My Google Calendar triggers this log message since setting it up:

```
starting. Version 1.1.3 in /opt/iobroker/node_modules/iobroker.calendar, node: v10.19.0
TypeError: Cannot read property 'val' of undefined
Updated calendar "TRS"
Got terminate signal. Checking desired PID: 0 vs own PID 27775
terminating
Terminated (START_IMMEDIATELY_AFTER_STOP): Without reason
Got terminate signal. Checking desired PID: 27816 vs own PID 27775
```

The above then repeats with PID 27816. "TRS" it the name of my calendar as per the CalDAV settings.

Possibly related to #15

**To Reproduce**  
1. Create instance
1. Add CalDAV URL pointing to a private Google Calendar
1. See error

**Expected behavior**  
No errors

**Versions:**  
 - Adapter version: 1.1.13
 - JS-Controller version: 2.2.9
 - Node version: 10.19.0
 - Operating system: Raspbian, ioBroker in docker

**Comment by WLAN-Kabel** [comment]
#15 The error occurred when a duration was specified instead of an end date. This should be fixed in version 1.1.3.

Could you turn on debug mode and provide me the output before the error occurs?

**Comment by agross** [comment]
If by debug mode you mean debug log (instance settings) this is all that was logged.

**Comment by WLAN-Kabel** [comment]
I think I found the mistake. I solved the bug in version 1.1.3, but only for CalDAV and not for iCal.

**Comment by agross** [comment]
Excellent. I'll be happy to help by testing the fix.
